### PR TITLE
Allow injection of a custom database path

### PR DESCRIPTION
**Pull request by g4vroche**

### CODE REVIEW

File: src/services/database.js [hunk 1]
```diff
@@ -4,19 +4,27 @@ import fs from 'fs';
...
+      // prepend path to filename
+      .map(filename => path.join.apply(null, [...paths, filename]))
+  );
+}
+
 const filenames = [
-  '../database.csv',
-  ...fs.readdirSync(path.join(__dirname, '../../data')),
+  path.join(__dirname, '../../database.csv'),
+  ...getFilePathList([__dirname, '../../data']),
```
Comment (rdubigny) [inline src/services/database.js]: according to the new function introduced, you should ever 

1. pass a single string as parameter and not an array
2. or keep the previous paradigm : generate an array of folder first, then pass it to you function

File: src/services/database.js [hunk 2]
```diff
@@ -4,19 +4,27 @@ import fs from 'fs';
 
 const database = { connection: null };
 
+export function getFilePathList(paths) {
```
[the remark this answers is not in the document]
Comment (g4vroche) [reply src/services/database.js]: Indeed, pushed a version with `getFilesPathsFromDir`

File: src/services/database.js [hunk 3]
```diff
@@ -4,19 +4,27 @@ import fs from 'fs';
...
+  return (
```
[the remark this answers is not in the document]
Comment (g4vroche) [reply src/services/database.js]: done

File: src/services/database.js (hunk 1 above)
Comment (g4vroche) [reply src/services/database.js]: Indeed, corrected  as well in the new PR.

**Pull request merged**